README.md

```diff
@@ -0,0 +1,111 @@
+## Rancher Feature Charts
```

Comment:
Nit: should we remove the "Feature"? I feel like we never formalize it and now we are moving towards just Rancher Charts

Reply:
Sure, that sounds good to me.

Reply:
updated in https://github.com/rancher/charts/pull/1677#discussion_r778466081